docs/CHANGELOG.asciidoc

```diff
@@ -8,25 +8,172 @@
+
+//tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-110830]]
+.Changes the `GET /api/status` default behavior
+[%collapsible]
+====
+*Details* +
+`GET /api/status` reports a new and more verbose payload. For more information, refer to {kibana-pull}110830[#110830].
+
+*Impact* +
+To retrieve the {kib} status in the previous format, use `GET /api/status?v7format=true`.
+====
+      
+[discrete]
+[[breaking-110738]]
+.Removes support for legacy exports
+[%collapsible]
+====
+*Details* +
+In {kib} 8.0.0 and later, the legacy format from {kib} 6.x is unsupported. For more information, refer to {kibana-pull}110738[#110738]
+
+*Impact* +
+Using the user interface to import saved objects is restricted to `.ndjson` format imports.
+====
+      
+[discrete]
+[[breaking-109896]]
+.Removes `map.regionmap.*`
+[%collapsible]
+====
+*Details* +
+The deprecated `map.regionmap.*` setting in kibana.yml has been removed. For more information, refer to {kibana-pull}109896[#109896].
+
+*Impact* +
+If you have maps that use `map.regionmap` layers:
+
+. Remove the `map.regionmap` layer.
+
+. To recreate the choropleth layer, use <<import-geospatial-data,GeoJSON upload>> to index your static vector data into {es}. 
+
+. Create a choropleth layer from the indexed vector data. 
+====
+      
+[discrete]
+[[breaking-109798]]
+.Removes `kibana.defaultAppId`
+[%collapsible]
+====
+*Details* +
+The deprecated `kibana.defaultAppId` setting in kibana.yml, which is also available as `kibana_legacy.defaultAppId`, has been removed. For more information, refer to {kibana-pull}109798[#109798].
+
+*Impact* +
+When you upgrade, remove `kibana.defaultAppId` from your kibana.yml file. To configure the default route for users when they enter a space, use the <<defaultroute,`defaultRoute`>> in *Advanced Settings*.
+====
+      
+[discrete]
+[[breaking-109350]]
+.Removes `courier:batchSearches`
+[%collapsible]
+====
+*Details* +
+The deprecated `courier:batchSearches` setting in *Advanced Settings* has been removed. For more information, refer to {kibana-pull}109350[#109350].
+
+*Impact* +
+When you upgrade, the `courier:batchSearches` setting will no longer be available.
+====
+      
+[discrete]
+[[breaking-108111]]
+.Removes `xpack.task_manager.index`
+[%collapsible]
+====
+*Details* +
+The deprecated `xpack.task_manager.index` setting in kibana.yml has been removed. For more information, refer to {kibana-pull}108111[#108111].
+
+*Impact* +
+When you upgrade, remove `xpack.task_manager.index` from your kibana.yml file.
+====
+      
+[discrete]
+[[breaking-108103]]
+.Removes dashboard-only mode
+[%collapsible]
+====
+*Details* +
+The legacy dashboard-only mode has been removed. For more information, refer to {kibana-pull}108103[#108103].
+
+*Impact* +
+When you upgrade, the legacy dashboard-only mode will no longer be available.
```

Comment:
This is a breaking change that is shared by the Kibana App team and the Kibana Platform Security team.

The App team made the changes to actually remove dashboard only mode in 8.0-alpha2.
The Platform Security team is going to submit changes for 7.16 to surface this in the upgrade assistant (see #111160), but at this point in time the upgrade assistant does not mention it.

Our intention is to surface a warning in the upgrade assistant when we detect that the `kibana_dashboard_only_user` role is being used, and let users know they can use Kibana Privileges to create an equivalent custom role instead.

With that in mind: do you think we should change this release note? It currently reads very bluntly.

Reply:
@jportner the content in this breaking change only applies to 8.0.0-alpha2, and  the 7.16.0-specific content should only appear in the 7.16.0 release notes. 

For 8.0.0-alpha2, is there a work around we can add?

Reply:
> @jportner the content in this breaking change only applies to 8.0.0-alpha2, and the 7.16.0-specific content should only appear in the 7.16.0 release notes.

:+1:

> For 8.0.0-alpha2, is there a work around we can add?

In 8.0+, users should use [Kibana privileges](https://www.elastic.co/guide/en/kibana/current/kibana-role-management.html#adding_kibana_privileges) to create an equivalent role that only has access to the Dashboard application within Kibana.